bentoml/_internal/server/service_app.py

```diff
@@ -289,7 +297,7 @@ async def api_func(
                     )
                 else:
                     response = JSONResponse("", status_code=e.error_code)
-            except Exception:  # pylint: disable=broad-except
+            except _StopIteration:
```

Comment:
What's the reason for this change?

Reply:
remove the pylint error

Reply:
we want to catch all the exceptions caused by the user function. It's necessary.

Reply:
@aarnphm let's change this back to `Exception`

Reply:
ok